### PR TITLE
Release 12.0.0

### PR DESCRIPTION
**Pull request by wyardley**

### CODE REVIEW

**Comment by bastelfreak** [comment]
we should try to include https://github.com/voxpupuli/puppet-rabbitmq/pull/899 in the release. do you have a few minutes to take a look? otherwise I might be able to work on it next Sunday.

**Comment by wyardley** [comment]
I’m not sure it’ll be obvious to me, so I’ll take a quick look; but otherwise may have to wait for you.

do you know what’s up with the arch failure? I’ll try re-running once.

**Comment by root-expert** [comment]
@wyardley can you rebase before merging this?

**Comment by wyardley** [comment]
@root-expert yes, I'm also updating the changelog / reference. Please re-review @root-expert @bastelfreak

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
...
+**Closed issues:**
+
+- Reference for log\_levels does not apply [\#894](https://github.com/voxpupuli/puppet-rabbitmq/issues/894)
+- Adding rabbitmq-delayed-message-exchange plugin [\#890](https://github.com/voxpupuli/puppet-rabbitmq/issues/890)
+- rabbitmqadmin will not be upgraded [\#804](https://github.com/voxpupuli/puppet-rabbitmq/issues/804)
+- fix or eliminate remaining "multiple expectations" warnings [\#603](https://github.com/voxpupuli/puppet-rabbitmq/issues/603)
+
+**Merged pull requests:**
+
+- Fix Bug \#804 'rabbitmqadmin will not be upgraded' [\#897](https://github.com/voxpupuli/puppet-rabbitmq/pull/897) ([s-johansson](https://github.com/s-johansson))
```
Comment (root-expert) [inline CHANGELOG.md:30]: I added bug label, please re-create the Changelog

Comment (wyardley) [reply CHANGELOG.md]: updated

**Review by root-expert — APPROVED** [review]

**Pull request merged**